src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
+ *
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const makeAnArray = sourceString.replace(/(\n|\r)/gm, '').split(';');
+  const cssObject = {};
+
+  makeAnArray.map((item) => {
```

Comment:
```suggestion
  makeAnArray.forEach((item) => {
```

better to use `forEach` to just  iterate through an array, 
`map` is usually used to create a new array